### DISCUSSION TITLE
Relative vs. absolute paths in Markdown documentation for rules

### DISCUSSION THREAD

**Discussion by stefankolb** [eslint/eslint#15271] [Q&A]
Building a version of the documentation that can be used offline, I stumbled upon some Markdown files having links to absolute paths/URLs instead of relative ones. For example, in `docs/rules/quotes.md`, line 160, there is a link to the `no-restricted-syntax` rule. Instead of having a relative link, i.e. `[no-restricted-syntax](no-restricted-syntax)`, it has a link to an absolute URL on the ESLint website, i.e. `[no-restricted-syntax](https://eslint.org/docs/rules/no-restricted-syntax)`.

So my question is: Is there a reason for these absolute links? If not, I could work on changing absolute links to relative ones.

**Comment by nzakas** [comment]
I don’t think relative links work correctly. If you use “semi”, that works when viewed on the website but not on GitHub; if you use “semi.md”, that works on GitHub but not on the website. The full URL ensures the link works in any location people view the file.
